### PR TITLE
Add new category: development browsers

### PR DESCRIPTION
**Pull request by Kilian**
Development browsers help designers and developers create websites faster, and offers a better way to test and validate designs. They're upcoming and I think they deserve their own category :) Full disclosure: I built Polypane.

### CODE REVIEW

**Comment by h3lad0** [comment]
😃

С уважением, Шикин Андрей

File: README.md [hunk 1]
```diff
@@ -373,6 +374,18 @@ Developers actively use version control tools for a long time, probably since 20
 
 </article>
 
+<article id="development-browsers">
+    
+### Development Browsers
+
+Development browsers have features that help developers and designers create and test websites and apps.
+
+* [Blisk](https://blisk.co) — provides a development workspace to develop and test modern web applications twice faster.
```
Comment (marcenuc) [inline README.md]: I think the correct link is https://blisk.io/

Comment (Kilian) [reply README.md]: You are right :flushed: I'll update it right away.

**Comment by valianka** [comment]
wow, it's great! Thank you soo much for adding a new section :)

**Pull request merged**